airsonic-main/src/main/java/org/airsonic/player/service/MediaFileService.java

```diff
@@ -542,13 +541,9 @@ private MediaFile createMediaFile(File file) {
                     }
 
                     // Look for cover art.
-                    try {
-                        File coverArt = findCoverArt(children);
-                        if (coverArt != null) {
-                            mediaFile.setCoverArtPath(coverArt.getPath());
-                        }
-                    } catch (IOException x) {
-                        LOG.error("Failed to find cover art.", x);
+                    File coverArt = findCoverArt(children);
+                    if (coverArt != null) {
+                        mediaFile.setCoverArtPath(coverArt.getPath());
```

Comment:
This seems more like a logic change that shouldn't be included with the more trivial changes.

Reply:
Not really: checkstyle complains about it, since the exception can't be raised anymore.

Reply:
Ah I see what you mean.